packages/app-project/components/ClassifyPage/components/ClassifierWrapper/ClassifierWrapperContainer.js

```diff
@@ -17,8 +17,30 @@ function storeMapper (stores) {
 @inject(storeMapper)
 @observer
 export default class ClassifierWrapperContainer extends Component {
+  constructor () {
+    super()
+    this.state = {
+      hasError: false
+    }
```

Comment:
Should we update Babel to allow the proposed syntax for instance properties?
```javascript
Class MyComponent extends Component {
  state = {
    stateVar: 'Initial state'
  }
…
}
```

Reply:
I'd be happy to, but we should probably make this a repo-wide change.